webtool/test_fetch.py

```
@@ -1010,3 +1010,36 @@ def einmal_scheitern(project, url, sprecher=None, sprache=None):
     # ausgeloest wuerde — er pruefte dann nur den ersten Versuch.
     assert versuche["n"] == 2, "der Wiederholungsversuch lief gar nicht — Test misst nichts"
     assert projekt_mod.datei_ansicht("Demo", "Mein Interview")["sprache"] == "en"
+
+
+def test_fetch_meldet_einen_LEEREN_bereich_VOR_dem_ersten_download(projekt, monkeypatch, capsys):
```

Ruff (0.16.2)

[warning] Unused function argument: `projekt`

(ARG001)